### PR TITLE
Updated Azure App Configuration Push Task Release notes

### PR DESCRIPTION
**Pull request by MaryanneNjeri**

### CODE REVIEW

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v4.4.0 - November, 15 2022
+* Added Import mode option support for KVSet profile.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:6]: I see "Import mode", "ImportMode", and "Import Mode" in this PR. Can we make up our minds about which format to go with? :)

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v4.4.0 - November, 15 2022
+* Added Import mode option support for KVSet profile.
+* Added Strict support for KVSet profile. With the added support of Strict for KVSet profile, when Strict is set to true any key-values in the store that are not included in the configuration file will be deleted.
+
+### v3.4.0 - October, 19 2022
+* Added ImportMode and Dry run options. Import Mode option is only supported for Default profile.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:10]: I would put import mode and dry run on two separate lines.

Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:10]: Can you write it in such a way that the property names that should be written into the yaml are clear?

File: releaseNotes/AzureDevOpsPushPipelineExtension.md (hunk 1 above)
Comment (MaryanneNjeri) [reply releaseNotes/AzureDevOpsPushPipelineExtension.md]: Apologies for that. Updated to ImportMode to be consistent with the way the property name should be written in YAML files

File: releaseNotes/AzureDevOpsPushPipelineExtension.md (hunk 2 above)
Comment (MaryanneNjeri) [reply releaseNotes/AzureDevOpsPushPipelineExtension.md]: Updated the property names "ImportMode" and "DryRun" to have them as they should be written in the yaml.

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v4.4.0 - November, 15 2022
+* Added ImportMode option support for KVSet profile.
+* Added Strict support for KVSet profile. With the added support of Strict for KVSet profile, when Strict is set to true any key-values in the store that are not included in the configuration file will be deleted.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:7]: "With the added support of Strict for KVSet profile, when"

It kind of repeats the previous sentence. Can you change that to "When using the KVSet profile and setting `Strict` to true ..."

**Review by jimmyca15 — APPROVED** [review]

**Pull request merged**